packages/switch/src/LionButtonSwitch.js

```diff
@@ -0,0 +1,95 @@
+import { LionButton } from '@lion/button';
+import { html, css } from '@lion/core';
+
+export class LionButtonSwitch extends LionButton {
+  static get properties() {
+    return {
+      readOnly: {
```

Comment:
Any reason we redefine this attr here? It's already added by LionButton (which shouldn't add it, because it's already part of HTMLElement ;))

Reply:
Should we even have a readonly? A switch is meant to immediately activate something. I would say it's either enabled or disabled. Readonly is for something that should be sent with a form, but you can't change. I'll check with UX.

Reply:
native button does not have readonly? not as attribute nor as property
also I assume readOnly is NOT part of HTMLElement?

did you maybe mean the FieldMixin or so? (which is not applied to a button?)